qzhou-embedding results

PennyYu123 opened the pull request:
<!-- Description of the PR goes here -->

### Checklist

- [ ] My model has a model sheet, report or similar
- [ ] My model has a reference implementation in [`mteb/models/`](https://github.com/embeddings-benchmark/mteb/tree/main/mteb/models) this can be as an API. Instruction on how to add a model can be found [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/reproducible_workflow.md) 
  - [ ] No, but there is an existing PR ___
- [ ] The results submitted is obtained using the reference implementation
- [ ] My model is available, either as a publicly accessible API or publicly on e.g., Huggingface
- [ ] I *solemnly swear* that for all results submitted I have not on the evaluation dataset including training splits. If I have I have disclosed it clearly.

 I have filled out the ModelMeta object to the extent possible
 I have ensured that my model can be loaded using
 mteb.get_model(model_name, revision) and
 mteb.get_model_meta(model_name, revision)
 I have tested the implementation works on a representative set of tasks.
 The model is public, i.e. is available either as an API or the wieght are publicly avaiable to download

PennyYu123 committed:
qzhou-embedding results

KennethEnevoldsen commented:
Hi @PennyYu123, I can neither find this model on Huggingface nor an implementation within the mteb package

Samoed commented:
@KennethEnevoldsen It was added in this PR https://github.com/embeddings-benchmark/mteb/pull/2965, but I can't find it on HF too

KennethEnevoldsen commented:
Given that the model submission was closed, I will also close this. Feel free to re-open if you resubmit the model

PennyYu123 commented:
I have corrected the wrong info in model_meta in the latest mteb PR and updated the model  implementation  code. Our Qingzhou model is now private and will be made public on HF tomorrow.

KennethEnevoldsen commented:
Ah, that is great. Do feel free to reopen the PR once the implementation has been reviewed
